recipes-security/selinux/selinux-autorelabel_%.bbappend

```diff
@@ -0,0 +1,13 @@
+#FILESEXTRAPATHS_append := "${THISDIR}/files:"
+
+do_install_append() {
+    sed -i -e 's#/.autorelabel#/var/aos/.autorelabel#g' ${D}${bindir}/${SELINUX_SCRIPT_SRC}.sh
```

Comment:
SELINUX_SCRIPT_SRC defined in other recipe. I'm not sure it will be visible here.

Reply:
this variable `SELINUX_SCRIPT_SRC` defined in `selinux-initsh.inc` which is require in `selinux-autorelabel` receipt